### ISSUE TITLE
exerslide build fails in new projects

### ISSUE THREAD

**Issue by lex93ushakov** [facebookincubator/exerslide#6]
`KDE Neon` (based on Kubuntu 16,04)

``` sh
sudo npm i -g exerslide-cli
```

``` sh
$ exerslide init presentation
module.js:328
    throw err;
    ^

Error: Cannot find module '/home/lex93ushakov/lab/exerslide/example/cli'
    at Function.Module._resolveFilename (module.js:326:15)
    at Function.Module._load (module.js:277:25)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at loadLocalCommands (/usr/local/lib/node_modules/exerslide-cli/bin/exerslide.js:80:20)
    at Liftoff.<anonymous> (/usr/local/lib/node_modules/exerslide-cli/bin/exerslide.js:53:5)
    at Liftoff.<anonymous> (/usr/local/lib/node_modules/exerslide-cli/node_modules/liftoff/index.js:202:12)
    at Liftoff.<anonymous> (/usr/local/lib/node_modules/exerslide-cli/node_modules/liftoff/index.js:169:7)
    at nextTickCallbackWith0Args (node.js:419:9)
    at process._tickCallback (node.js:348:13)
```

**Comment by fkling** [comment]
It seems that exerslide-cli (or rather liftoff) thinks that the exerslide package is installed in `/home/lex93ushakov/lab/exerslide/example/`. What is the contents of that folder? Are you running `exerslide init` in an empty folder?

**Comment by fkling** [comment]
Oh... did you clone this repository on run `exerslide init` inside `exerslide/example`? Why? You don't have to clone this repo to use exerslide. And you shouldn't be running `exerslide init` in the `example/` folder.

**Comment by lex93ushakov** [comment]
http://asciinema.org/a/b5at3i2tmhl9mm2dk5nv43nid

**Comment by lex93ushakov** [comment]
Init successfully bootstrapped in clean directory, but `exerslide build` command fails (look at my previous post).

**Comment by fkling** [comment]
Confirmed, will investigate. Thanks!

**Comment by fkling** [comment]
This is fixed with via 84c6aec, bcc9c00, c839be2 and 986e07b.
